Source/lib/maxicode/MaxiCodeReader.cs

```diff
@@ -66,7 +66,7 @@ public Result decode(BinaryBitmap image, IDictionary<DecodeHintType, object> hin
             if (decoderResult == null)
                 return null;
 
-            var result = new Result(decoderResult.Text, decoderResult.RawBytes, NO_POINTS, BarcodeFormat.MAXICODE);
+            var result = new Result(decoderResult.Text, decoderResult.Data, decoderResult.RawBytes, NO_POINTS, BarcodeFormat.MAXICODE);

```

Comment:
should be reverted, see the comment at DecodedBitStreamParser.cs; Result class can generate the value of Data property if the reader doesn't give a better value.